prow/config.yaml

```diff
@@ -776,16 +715,12 @@ presubmits:
         volumeMounts:
         - name: docker-graph
           mountPath: /docker-graph
-        ports:
-        - containerPort: 9999
-          hostPort: 9999
         resources:
           requests:
             memory: "6Gi"
       volumes:
       - name: docker-graph
-        hostPath:
-          path: /mnt/disks/ssd0/docker-graph
+        hemptyDir: {}
```

Comment:
extra 'h'

Reply:
hah! thanks. somewhat worrying that we don't automatically validate this.

Reply:
fixed